mlir/include/mlir/Dialect/MIOpen/MIOpenOps.td

```diff
@@ -204,8 +205,8 @@ def MIOpen_BlockwiseCopyOp:
     MIOpen_Op<"blockwise_copy">,
     Arguments<(ins AnyMemRef:$source,
                    AnyMemRef:$dest,
-                   AnyMemRef:$sourceCoord,
-                   AnyMemRef:$destCoord,
+                   VectorOfLengthAndType<[3], [I32]>:$sourceCoordVector,
```

Comment:
Why lengths is 3 instead of 2?

Reply:
take group convolution into consideration.